Extension/package.json

```diff
@@ -71,7 +71,7 @@
             "null"
           ],
           "default": null,
-          "description": "The full path of the clang-format executable.",
+          "description": "The full path of the clang-format executable. Changing this setting requires a Reload Window if it contains environment variables.",
```

Comment:
did it not work to intercept the didChangeConfigurations message in protocallFilter.ts?

Reply:
I didn't investigate that. I can.

Reply:
It doesn't appear possible to "replace" a setting in that filter, but I can change it to ignore that particular setting change and send our own message with the correct setting.

Reply:
Yes, I think that would be better.  Thanks!